bitmap.js

```diff
@@ -0,0 +1,39 @@
+'use strict'; 
+
+const fileHelper = require('./lib/bitmap-file-helper.js');
+const Bitmap = require('./model/bitmap-constructor.js');
+
+
+let filesToConvert = [`${__dirname}/img/palette-bitmap.bmp`, `${__dirname}/img/optimus-prime.bmp`];
+transformBMP(filesToConvert);
+
+function transformBMP(files) {
+  let colorTransforms = [];
+  if (process.argv.length > 2) {
+    for (let i = 2; i < process.argv.length; i++) {
+      if (Bitmap.colorTransforms.includes(process.argv[i])) {
+        colorTransforms.push(process.argv[i]);
+      }
+    }
+    return fileColorTransform(files, colorTransforms);
+
+  }
+  colorTransforms = ['invertTransform', 'grayTransform', 'blueTransform'];
+
+  return fileColorTransform(files, colorTransforms);
+
+
+  function fileColorTransform(files, colorTransforms) {
```

Comment:
@brials @remilonwheels @caylazabel might be best to pull this color transformer function out of your `transformBMP` function for reusability cleanliness and readability